src/pages/supplements/Sofs.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from "react"; 
+import { Button, ButtonGroup, Form, Label, Input, } from 'reactstrap'; 
+
+const Sofs = (props) => {
+    const [sofsSupplement, setSofsSupplement] = useState({budgetDescription: null, budgetFile: null}); 
+
+    return (
+        <Form>
+            <Label for="budgetDescription">Budget Description</Label>
+            <Input type="textarea" id="budgetDescription" onChange={text => setSofsSupplement(prev => ({...prev, budgetDescription: text}))}/>
```

Comment:
So I did some digging to see how I would display the supplements in the review page, and found out that this change handler and the travel insurance change handler don't correctly save the values of the supplements. 
![image](https://user-images.githubusercontent.com/21688954/67254496-f5c72a00-f442-11e9-87c2-1602ee677254.png)

The supplements are saving event objects rather than strings for the events, which is going to make it hard to display them on the review page. The reason is because onChange handlers accept a JavaScript `event` and not the event's value. To fix this, you'll have to have to change `budgetDescription : text` to `budgetDescription : text.target.value` (where `text` is the `event` emitted by `onChange`). I did something similar in the basic info page, if you want to use that as a reference!